media-source-respec.html

```diff
@@ -2741,6 +2860,25 @@ <h3>{{HTMLMediaElement}}.{{HTMLMediaElement/buffered}}</h3>
           </li><li>Return the current value of this attribute.</li>
         </ol>
       </section>
+
+      <section id="htmlmediaelement-extensions-srcobject">
+        <h3>{{HTMLMediaElement}}.{{HTMLMediaElement/srcObject}}</h3>
+
+          <p>If a {{HTMLMediaElement}}.{{HTMLMediaElement/srcObject}} attribute is assigned a {{MediaSourceHandle}},
+            then set {{MediaSourceHandle/[[has ever been assigned as srcobject]]}} for that {{MediaSourceHandle}} to
+            true as part of the synchronous steps of the extended {{HTMLMediaElement}}'s {{HTMLMediaElement/srcObject}}
+            setter that occur before invoking the element's load algorithm.
+            <p class="note">This prevents transferring that {{MediaSourceHandle}} object ever again, enabling clear
+              synchronous exception if that is attempted. Rather than risking interoperability of specifying some more
+              complex behavior in this scenario, such as causing media element error, a synchronous exception on
+              attempted transfer of a handle that has ever been assigned to any {{HTMLMediaElement}}'s
+              {{HTMLMediaElement/srcObject}} attribute is simpler and provides rapid indication to the API user of the
+              transfer failure.</p>
```

Comment:
I agree with the note, but the specification doesn't have to justify itself :-)

Reply:
Ack. Extra bits of justification removed, leaving only basic reason why this slot is needed. Done in c62ebd6